ops/Makefile

```diff
@@ -16,7 +16,7 @@ CURL_TIMEOUT?=20
 	@case $* in dev|stg|prod|prod-dmz|global) ;; *) echo "$* is not a valid environment"; exit 1;; esac
```

Comment:
What is our `global` environment?  Do we still need this?

Reply:
`global` is the environment where Octopus Deploy lives. It is very much needed.